astropy/coordinates/builtin_frames/teme.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+Contains the definition of the TEME frame.
+
+TEME is a True equator, Mean Equinox coordinate frames used in NORAD TLE
+satellite files.
```

Comment:
Probably worth noting here (or somewhere else in the docstring) that this is specifically aimed at following the Vallado et al (2006) convention?  I think it's still reasonable to call it "the" TEME, but in case someone does a re-hash of Vallado 06 or something in the future, it's good to be explicit that that's implicitly what the frame is based on.